engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.controller.js

```diff
@@ -63,6 +63,10 @@ angular.module('Bastion.capsule-content').controller('CapsuleContentController',
             }
         }
 
+        $scope.fetchUrl = function (cvId) {
+            return cvId === 1 ? '/products' : '/content_views/' + cvId + '/versions';
```

Comment:
I think i mentioned this on irc, but putting it here:  Checking for id==1 will only work for the first org :)  there should be a 'default' attribute you can use

Reply:
oh gotcha, didn't know it would only work for the first organization.  I'll update.